.gitignore

```diff
@@ -5,6 +5,7 @@ theme-dev-utils/
 theme-dev-utils
 vendor/
 *.DS_Store
+.DS_Store
```

Comment:
I'm hoping this will stop any more `.DS_Store` files being added 🤞

Reply:
Should CBT also remove these?

Reply:
Yeah it could do, maybe that's a nice thing to add for preparing a theme export file. Although, if the theme was exported on macOS and edited somehow, and then compressed again manually, the files would probably return.

I think it would be easy/quick to add to CBT, so it might be worth adding anyway.